content/en/docs/concepts/architecture/leases.md

```diff
@@ -29,7 +29,7 @@ This is used by control plane components like `kube-controller-manager` and `kub
 HA configurations, where only one instance of the component should be actively running while the other
 instances are on stand-by.
 
-## kube-apiserver identity
+## API Server Identity
```

Comment:
```suggestion
## API server identity
```
or
```suggestion
## kube-apiserver identity {#api-server-identity}
```
(essentially: retain the original). Only kube-apiserver implements this mechanism as far as I know.

Reply:
All API servers with storage are expected to implement this (i.e. aggregated API servers and Kube API server).  This is currently broken outside of KAS though.

Reply:
> OK. Presumably the expectation only comes in once the feature is GA?

This was something we missed by accident.  It should have been part of the beta.